### DISCUSSION TITLE
Custom shortcut ⌘F1 not allowed

### DISCUSSION THREAD

**Discussion by invisibilitys** [MonitorControl/MonitorControl#702] [Q&A] [answered]
Hi 

I've been trying to set brightness controls to ⌘F1 and ⌘F2. I get the error "This keyboard shortcut cannot be used as it's already a system-wide keyboard shortcut." when I try to use ⌘F1, but not ⌘F2. However I've checked my system-wide keyboard shortcuts, and nothing seems to be using ⌘F1 at all, because I can set a different shortcut to ⌘F1 in System Preferences when it should give me an error if something is already using it. 

Another thing is when I do set a system-wide keyboard shortcut to ⌘F1, and then try to set ⌘F1 as a custom shortcut on Monitor Control, there is no feedback of "This keyboard shortcut cannot be used as it's already a system-wide keyboard shortcut.". 

Not sure if there is something special about ⌘F1 in Monitor Control or something I can't see in my computer using ⌘F1 haha

**Comment by waydabber** [comment, marked as answer]
Hi @invisibilitys,

afaik command+F1 is a system wide shortcut for display mirroring on mac notebooks. It is not a really important one so it could be overwritten I agree. However custom shortcuts are actually handled by [KeyboardShortcuts](https://github.com/sindresorhus/KeyboardShortcuts), so you should post this question there.

**Reply by L-i-v-y** [reply]
Sorry, just posted an issue about this before seeing the hint for checking the discussions tab (maybe add this to the issue template, next to the wiki hint) — do you mean the display mirroring option in the menu bar or somewhere else? Just wondering since it seems to work fine with Lunar and doesn't appear to do anything for me when I just press the combination – even if there is an AirPlay receiver present in my network?
